ohio-university-student-web-developers/ou-drupal-extension
# Strict

CHANGES:
- CKE specific classes or attributes that do not go into the actual source for the page are no longer removed
- 'use strict' is now applied to all JavaScript files

HOW TO TEST:
- Run "Clean HTML" on pages and make sure flags or tables do not appear to be removed.
- Try out many different parts of the extension to break it.
